opba-protocols/hbci-protocol-tests/hbci-bdd-sandbox/src/test/java/de/adorsys/opba/protocol/hbci/tests/e2e/sandbox/HbciSandboxConsentE2EHbciProtocolTest.java

```diff
@@ -236,6 +252,49 @@ void testDirectTransactionListWithoutSca() {
                 );
     }
 
+    @Test
+    void testAccountsListCacheUpdate() {
+        testAccountsListWithConsentNoScaButUserHasOneSca();
```

Comment:
Add assertion
`verify(onlineBankingService, times(1)).loadBankAccounts(any());`
right after call, as there can be 0 invocations there which breaks test ides